libs/testing/e2e/src/lib/support/email-account.ts

```diff
@@ -0,0 +1,171 @@
+// use Nodemailer to get an Ethereal email inbox
+// https://nodemailer.com/about/
+import { createTestAccount } from 'nodemailer'
```

Comment:
This no longer works. See [this Slack thread](https://islandis.slack.com/archives/C03HC3VTY12/p1730199322242049) for more info.

> ethereal.email is not allowing inbound emails anymore
> https://ethereal.email/help#inbound

Reply:
Addressing in https://github.com/island-is/island.is/pull/16816